general/mail_control.py

```diff
@@ -8,6 +8,10 @@
 
 from userdetails.models import User
 
+"""
+Deprecated: the method User.send_email() is simpler and can replace the functions in this module.
+"""
```

Comment:
Understandable, though I will miss the small html markup that was used on the emails

Reply:
Don't worry, I won't get rid of the nice HTML emails that we already have :smile:. I'm just not bothering with writing them myself. But if you want to add them I won't mind.

It's just that HTML emails are so difficult to write because they render differently in different e-mail clients. So you need to test them in all major clients. And short to the point e-mails are also easier to read.